lua/autorun/nl_library.lua

```diff
@@ -0,0 +1,607 @@
+			timer.Remove("NLLSleepIn"..id)
+			timer.Create("NLLSleepIn"..id, newdelay, 1, function()
+				NLL.Ragdoll(ply, pushdir,stuntype)
+				ply.nllismuted = true
+				timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+					NLL.UnRagdoll(ply)
+					ply.nllismuted = false
+				end)
+			end)
+			
+			ply.DrowsyDuration = newdelay
+			net.Start("nllsetdrowsy")
+			net.WriteInt(newdelay,16)
+			net.Send(ply)
+			return
+		end
+		timer.Create("NLLSleepIn"..id, timebefore, 1, function()
+			NLL.Ragdoll(ply, pushdir,stuntype)
+			ply.nllismuted = true
+			timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+				NLL.UnRagdoll(ply)
+				ply.nllismuted = false
+			end)
+		end)
+		ply.DrowsyDuration = timebefore
+		net.Start("nllsetdrowsy")
+		net.WriteInt(timebefore,16)
+		net.Send(ply)
+		return
+	end
+	if stuntype == "STN" or stuntype == "TZE" then // Taze shakes more violently than Stun
+		if not ply.nllstnpr then ply.nllstnpr = 0 end
+		ply.nllstnpr = ply.nllstnpr + timebefore // Stun Damage
+		if ply.nllstnpr < 100 then // Not above 100 Stun Damage, don't do anything.
+			if timer.Exists("NLLSTNHeal"..id) then timer.Remove("NLLSTNHeal"..id) end
+			timer.Create("NLLSTNHeal"..id,15,1,function()
+				ply.nllstnpr = 0 // They haven't been hit by stun for 15 seconds, heal that stun damage
+			end)
+			return
+		end
+		ply.nllstnpr = 0
+		NLL.Ragdoll(ply, pushdir,stuntype)
+		ply.nllismuted = true
+		timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+			NLL.UnRagdoll(ply)
+			ply.nllismuted = false
+		end)
+		return
+	end
+	if stuntype == "---" then // "Surrender". Supposed to allow self ragdolling, unused for now.
+		NLL.Ragdoll(ply, pushdir,"---")
+		ply.nllismuted = false // You're still awake, you shouldn't be muted while surrendering.
+		timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+			NLL.UnRagdoll(ply)
+			ply.nllismuted = false
+		end)
+		return
+	end
+	if stuntype == "PLV" then // Pulverize, bone anihilation method
+		NLL.Ragdoll(ply, pushdir,"PLV")
+		ply.nllismuted = true
+		timer.Create("NLLUnragdoll"..id, timedur, 1, function()
+			NLL.UnRagdoll(ply)
+			ply.nllismuted = false
+		end)
+		return
+	end
+end
+else // clientside code
+
+condcolors = { // Will add the ability to customize condcolors later
+	["ZZZ"] = {Color(150,150,255),"sleeping","wake up"},
+	["STN"] = {Color(255,150,150),"stunned"},
+	["GAY"] = {Color(255,0,255),"gay"},
+	["PLV"] = {Color(255,0,0),"suffering","make it stop"},
+	["---"] = {Color(150,150,150),"surrendering","make at ease"}
+}
+
+function NLL_BuddhaKillUpdate(ply,cmd,args)
+	if args[1] != "0" then
+		net.Start("nllupdatebuddha")
+		net.WriteInt(1,4)
+		net.SendToServer()
+		return
+	end
+	net.Start("nllupdatebuddha")
+	net.WriteInt(0,4)
+	net.SendToServer()
+	return
+end
+
+concommand.Add( "ttt_nl_buddhakill", function(ply,cmd,args) NLL_BuddhaKillUpdate(ply,cmd,args) end, nil, "If nonzero, covertly killing will instead set your victim's hp to 1", 0 )
```

Comment:
Use a Convar with userinfo enabled and `ply:GetInfoNum()` for this instead of concommand. You don't need to manually network it and can get rid of the net message entirely.